MAINTAINERS.md

```diff
@@ -0,0 +1,10 @@
+# Maintainers
+
+Maintainership is on a per project basis.
+
+Core-maintainers
+  - [derekcollison](https://github.com/derekcollison)
```

Comment:
Should we make these normal names since we are linking? What I mean is
[Derek Collison](https://github.com/derekcollison)

Reply:
Let's try this

Derek Collison <derek@nats.io> [@derekcollison](https://github.com/derekcollison)